app/controllers/account_activation_controller.rb

```diff
@@ -0,0 +1,26 @@
+class AccountActivationController < ApplicationController
+  before_action :load_user, only: :edit
+
+  def edit
+    if @user && !@user.activated? && @user.authenticated?(:activation, params[:id])
```

Comment:
cần `if @user` ko cu =)) 
```suggestion
    if !@user.activated? && @user.authenticated?(:activation, params[:id])
```